Gulpfile.js

```diff
@@ -171,6 +170,24 @@ gulp.task("build-check-test", gulp.series(
 gulp.task("build-lint-test", gulp.task("build-check-test"));
 
 // Helper functions:
+function spawnTask(command, done, args)
+{
+    let lCP;
+    if (args !== undefined)
+    {
+        lCP = spawn(command, args, { stdio: "inherit" });
+    } 
+    else
+    {
+        lCP = spawn(command, { stdio: "inherit" });
+    }
+    
+    lCP.on("error", done);
```

Comment:
Isn't the error case handled by `on("close")`? Also, won't this trigger a double exit / skip the special handling of the on close callback?

Reply:
```suggestion
```

Reply:
https://nodejs.org/api/child_process.html#event-error

You're right there's a potential double exit situation. I've adjusted accordingly;

Reply:
From the docs you linked, I would assume `on(close)` covers us completely. I've pushed a change to use it as such, if you can provide a counter-example or describe your concern with dropping the `on(error)` we can revist

Reply:
Okay, looking at your code locally, I can see that you are trying to pipe the `error` object if one exists, else fallback to codes. This is good, as more info is generally better (even if it costs a bit of function, complexity). I'm okay with this, you can resolve